src/fhirtypes/StructureDefinition.ts

```diff
@@ -596,3 +600,25 @@ const PROPS = [
   'baseDefinition',
   'derivation'
 ];
+
+const PRIMITIVES = [
```

Comment:
If this list comes from FHIR documentation, it would be useful to link to that documentation here.

Reply:
If we only use this list to check if something is a primitive, then I *think* we can just test to see if the first letter is lowercase.  I'm pretty sure that is a reliable convention for checking on primitiveness in FHIR.

If we do actually need a complete list of primitive names, then I think maybe we should consider dynamically building this list from the FHIR definitions instead.  That should be possible and it would ensure that we reflect any new/different primitives between FHIR versions.